src/main/java/me/profelements/dynatech/items/electric/AntigravityBubble.java

```diff
@@ -19,16 +20,11 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.HashSet;
-import java.util.Iterator;
 import java.util.List;
-import java.util.Set;
-import java.util.UUID;
 
 public class AntigravityBubble extends AMachine {
 
-    private final Set<UUID> enabledPlayers = new HashSet<>();
-
+    private static Location bubbleLocation;
```

Comment:
This is a static field but you are setting it each tick and referencing it onBreak which means a different block could tick then you break this one and it all goes wrong. This wont work I'm afraid. You normally need a Map<Location, Object> of some sort when caching things to a SlimefunItem/Machine/Block etc.